bats_test/generate-wallet/generate-wallet.bats

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bats
```

Comment:
Thank you for your PR @xloem 👍 

As you're adding BATS for `generate-wallet` can we add one to test when it failed?
Example:
`yarn ardrive generate-wallet -s 'invalid seed phrase'`
It should failed with the following message: `Error: 'invalid seed phrase' is not a valid 12 word seed phrase!`